kvm-forum-hackfest-2017/README.md

```diff
@@ -0,0 +1,41 @@
+# KVM Forum KubeVirt Workshop 2017
+Have a room at KVM Forum KubeVirt Workshop 2017 to allow people interested in
+KubeVirt to sit together and talk.
+There will be a few sessions to have a little guidance/structure.
```

Comment:
FYI in terms of timing, the KVM Forum runs  Wed 24th -> Fri 26th, with the Wed being a joint-day in the LinuxCon track, the Thur/Fri being KVM exclusive. I'm travelling to Brno on Sat for further Red Hat internal meetings on Sun/Mon afterwards. If we want a 2-day workshop, I'd suggest partial overlap Tue 23rd/Wed 24th. Or a 1 day workshop, try Tue 23rd